cmd/network.go

```diff
@@ -73,7 +75,20 @@ func produceNetworkCmdRunFunc(o networkOperator, action int) cmdRunFunc {
 				log.Fatalf("Removing Network failed: %s", err)
 			}
 		}
-
+	case networkCreateAction:
+		return func(cmd *cobra.Command, args []string) {
+			ctx := context.Background()
+			cNetwork, err := client.CreateNetwork(ctx, gsclient.NetworkCreateRequest{
+				Name: args[0],
+			})
+			if err != nil {
+				log.Error("Create network has failed with error", err)
+				return
```

Comment:
Hmm, no. While this does log to stderr it does not make process exit with status set to non-zero.

Reply:
Oh good catch, i need to change this thanks 👍